app/assets/stylesheets/application.scss

```diff
@@ -12,4 +12,11 @@
  *
  *= require_tree .
  *= require_self
+ *= require trix
  */
+@import "bootstrap";
+
+body{
+  background-image: none;
+  background-color: #f2f7ff;
+}
```

Comment:
can you fix your editor and force it to add new lines at the end of file?

Reply:
Added this configuration to my sublime text editor, fixed problems

{
  "ensure_newline_at_eof_on_save": true,
  "trim_trailing_white_space_on_save": true,
  "translate_tabs_to_spaces": true,
  "tab_size": 2,
  "rulers": [80]
}